### PR TITLE
Add custom bootstrap from CDN

### PR DESCRIPTION
**Pull request by johnmdonahue**
:warning: WIP: do not have VS guessing something else needs to be changed when assets are removed :warning:

### CODE REVIEW

**Comment by johnmdonahue** [comment]
@ardiesaeidi can you review these changes?

**Comment by johnmdonahue** [comment]
@hattan: @ardiesaeidi is ignoring me can you review?

File: my-website/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -4,6 +4,9 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>@ViewBag.Title - My ASP.NET Application</title>
+    <link rel="stylesheet" href="//ui.edgecast.com/libs/font-awesome/4.2.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="//ui.edgecast.com/styles/0.0.4/css/core.min.css" />
+
```
Comment (githubstudent) [inline my-website/Views/Shared/_Layout.cshtml]: This is nice! i love customization. Are these URLs publicly accessible ?

Comment (johnmdonahue) [inline my-website/Views/Shared/_Layout.cshtml]: Indeed :)

Comment (githubstudent) [inline my-website/Views/Shared/_Layout.cshtml]: :+1:

**Comment by githubstudent** [comment]
this all looks good to me. Huge fan of customization

**Pull request merged**